### DISCUSSION TITLE
Is it possible to embed wasm into libv86.js?

### DISCUSSION THREAD

**Discussion by dmgnr** [copy/v86#779] [Ideas]
Yes https://guido.io/posts/embedding-webassembly-in-javascript/
Will this be compatible with libv86.js?

**Comment by dmgnr** [comment]
Actually the secret~~(not anymore)~~ Im just in a fight with my friend about making any cool single js file nodejs program

Also can i also embed iso and bios...

**Comment by copy** [comment]
You should be able to do `new V86({ wasm_fn: async env => (await WebAssembly.instantiate(bytes, env)).instance.exports })` or similar.

**Reply by dmgnr** [reply]
It worked, thanks!

**Comment by dmgnr** [comment]
Is there a way to load a module from a buffer?? currently im using this way :
write using fs
require in script
delete using fs
Because currently i am merging libv86.js and nodejs.js with closure compiler and it doesnt seem to work, it still require ./libv86.js, I tried to get the export, tried to redirect the export, but none worked...
